Packs/Forescout/Integrations/Forescout/Forescout.py

```diff
@@ -513,7 +524,8 @@ def http_request(method: str, url_suffix: str, full_url: str = None, headers: Di
 
         error_class = str(e.__class__)
         err_type = '<' + error_class[error_class.find('\'') + 1: error_class.rfind('\'')] + '>'
-        err_msg = f'\nERRTYPE: {err_type}\nERRNO: [{e.errno}]\nMESSAGE: {e.strerror}\n' \
+        err_msg = f'\nERRTYPE: {err_type}\nERRNO: [{e.errno if "errno" in dir(e) else None}]\n' \
+                  f'MESSAGE: {e.strerror if "strerror" in dir(e) else str(e)}\n' \
```

Comment:
What is this fix?

Reply:
Sometimes original exception in exception chain or the casting returns an error which does not have an errno and strerror attributes. If so, it will not fail again here on AttributeError but will print the error it received with str(e). This happened to me when trying to configure our instance and it could hide the real errors.